crates/orchestrator/src/generators/command.rs

```diff
@@ -9,6 +9,7 @@ pub struct GenCmdOptions<'a> {
     pub relay_data_path: &'a str,
     pub use_wrapper: bool,
     pub bootnode_addr: Vec<String>,
+    pub use_default_ports_in_cmd: bool,
```

Comment:
I don't really understand what it means, aren't we suppose to fallback on default port if no port are provided anyway ?

Reply:
Not really, because we `always` set the ports (randomly is the user don't set). So, at this point we will always have port assigned and that why we need to branch to expose the ports that the `image` are exposing. Those will be later using for port-forwarding to the local (randomly selected) ports.
make sense?